site/styles/_typography.scss

```diff
@@ -0,0 +1,116 @@
+small {
+  font-size: 85%;
+}
+
+a:not([class]) {
+  text-decoration: underline;
+}
+
+p {
+  font-size: 16px;
+  font-weight: $font-weight-light;
+  letter-spacing: 0.5px;
+  line-height: 23px;
+}
+
+h1 {
+  color: var(--body-color);
+  font-size: 35px;
+  font-weight: $font-weight-normal;
+  letter-spacing: 0.8px;
+  margin-bottom: 8px;
```

Comment:
I don't think structural CSS (in this case `margin`) should be in a file dedicated to typography.

Reply:
Note to self: we decided that this isn't a huge deal in the elab.